elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -398,7 +399,73 @@
     margin: 0;
     padding: 0;
     font-size: 16px;
+    color: var(--pfe-theme--color--ui-complement,#464646);
   }
+
+  h1 a,
+  h2 a,
+  h3 a,
+  h4 a,
+  h5 a,
+  h6 a {
+    color: var(--pfe-theme--color--ui-complement,#464646);
+  }
+
+  li a {
+    color: var(--pfe-broadcasted--link,#06c);
+    &:hover {
+      color: var(--pfe-broadcasted--link--hover,#036);
+      text-decoration: underline;
+    }
+  }
+}
+
+.pfe-navigation__dropdown--single-column {
+  // @todo animation and polish
+  display: none;
+
+  @include breakpoint('nav-break') {
+    position: absolute;
+    top: 100%;
+    left: auto;
+    width: auto;
+    max-width: 100%;
+    min-width: 32px;
```

Comment:
@wesruv that needs to be changed, I just had that in there initially for testing purposes, so we can probably just remove that rule. Do you think we need a min-width setting at all since we are letting it be as wide as the widest link in the column?

Reply:
I think it'd look goofy if the dropdown was skinnier than the menu item it's coming from, so I'd probably set a min width that's a decent size to try and prevent that from happening